### PR TITLE
Notification fixes.

### PR DESCRIPTION
**Pull request by sri2k1us**
- Fix issue where notification doesn't show link for VICE running notification
- Fix issue where notification doesn’t highlight new notifications
- Fix iPhone 5 - text overflow on timestamp
- Add retry on error and Fix Error handling when fetching notifications
- Fix case where there are no notifications for user 

- No Notifications
image

- Highlight New notifications / VICE Link
image

- iPhone 5 / VICE Link / Repositioned buttons for mobile
image

- Fetch error
image

### CODE REVIEW

**Review by ash3rz — APPROVED** [review]
LGTM :shipit: 
I only had minor comments not necessary before merging

File: src/components/notifications/NotificationsMenu.js [hunk 1]
```diff
@@ -60,9 +89,17 @@ function NotificationsMenu(props) {
...
+                if (results?.unseen_total > 0) {
+                    setUnSeenCount(results?.unseen_total);
+                }
+                setError(null);
             },
+            onError: setError,
+            retry: 3,
+            //copied from react-query doc. Add exponential delay for retry.
+            retryDelay: (attempt) =>
+                Math.min(attempt > 1 ? 2 ** attempt * 1000 : 1000, 30 * 1000),
```
[the remark this answers is not in the document]
Comment (sri2k1us) [reply src/components/notifications/NotificationsMenu.js:102]: Cool. In that case I can remove this.

**Comment by sri2k1us** [comment]
thanks for the review.

File: src/components/notifications/NotificationsMenu.js [hunk 2]
```diff
@@ -31,10 +41,29 @@ function getTimeStamp(time) {
...
+function InteractiveAnalysisUrl(props) {
+    const { t } = useTranslation(["common"]);
+    return (
+        <span>
+            {". "}
+            <Link
+                href={props.notification.payload.access_url}
+                target="_blank"
+                rel="noopener noreferrer"
+            >
```
Comment (psarando) [inline src/components/notifications/NotificationsMenu.js:53]: Don't forget about the `ExternalLink` util.

**Review by psarando — COMMENTED** [review]
LGTM 👍

**Pull request merged**